prime-router/src/main/kotlin/azure/Event.kt

```diff
@@ -103,20 +101,9 @@ abstract class Event(val eventAction: EventAction, val at: OffsetDateTime?) {
         fun parseQueueMessage(event: String): Event {
             // validate incoming queue message is in the expected format. This will error out with an
```

Comment:
Does this comment still applicable?

Reply:
something not in AC:

Event.parseQueueMessage(...)

call JacksonMapperUtilities.defaultMapper.readValue(event) without catch checked exceptions, they will bubble up to the callers, and callers handle them in various ways - most cases, the parsing exceptions info are not properly logged to provide as much info as possible to trouble shooting.

created #12749

Reply:
Yes, the comment is still applicable. 

If there is an exception here, it would be the fault of developers not the fault of the person submitting the data which is why I think we aren't bubbling it up to them. It would be in the logs though.